### ISSUE TITLE
restart policy?

### ISSUE THREAD

**Issue by maci0** [htdvisser/taiga-docker#12]
Might be good to add   restart: always for the containers.

**Comment by htdvisser** [comment]
As far as I know this option is left out in many other projects as well. The `taiga.yml` file is meant for allowing users to get a quick preview of the system, not for actual installations. As soon as you want to keep Taiga running for a longer time, you'll want to customise the `.yml` anyway.

**Comment by hanu18** [comment]
Hi buddy,

Could you please add proper steps for installing, running and using this image.
Existing document is very poor and confusing for new bees in docker and taiga.

I have started all containers on my ubuntu server now how can i use taiga using my server IP or hostname in any browser ?? :(

Using $docker ps I can see all containers all up and runing.
I am using poderosa Once i run taiga-back container I need to open new window to start fron container.

In short please provide complete document to make it usable.
Thanks

**Comment by htdvisser** [comment]
Hello @hanu18,

A good starting point for Docker is the [Docker Documentation](https://docs.docker.com/).

I created a very simple starting point for this image using `docker-compose`. You can find the template [here](https://github.com/htdvisser/taiga-docker/blob/stable/taiga.yml). If you save that file as `docker-compose.yml` and change the `hostname`, `SECRET_KEY` and `EMAIL_*` variables to match your environment, everything should work after a `docker-compose up`. The only thing you have to do manually is initializing the database as explained in the [README](https://github.com/htdvisser/taiga-docker/tree/stable/taiga-back#database-initialization).
